Core/Goals/FollowRouteGoal.cs

```diff
@@ -153,7 +154,7 @@ public override async Task PerformAction()
                 {
                     logger.LogError("wtf too far away kekw");
                     routeToWaypoint.Pop();
-                    return;
+                    // return;
```

Comment:
There's a hard limit in the `PathingAPI`, if the distance is higher then 100 yard it wont return any path.

Reply:
It causes crashes of the bot. Pathfinding sometimes works really bad. I faced with an issue where bot stayed about 5 yards away from the path point and the player just stayed or jumping without any movement. In log, I saw the message from this function. When I commented this line, the bot was able to get back into the path.